ros2service/ros2service/verb/call.py

```diff
@@ -59,7 +59,7 @@ def main(self, *, args):
 def requester(service_type, service_name, values, period):
     # TODO(wjwwood) this logic should come from a rosidl related package
     try:
-        package_name, srv_name = service_type.split('/', 2)
+        package_name, _, srv_name = service_type.split('/', 3)
```

Comment:
Instead of ignoring the second value this could consolidate the `middle_module` below?

Reply:
with that new logic, I believe we don't need that `middle_module` logic any longer. @sloretz do you think this could address the TODO here?

As far as I understand, this new type identification serves exactly the case of separating actions from services, right? So storing the second value of the split call as the middle module should suffice.

Reply:
> with that new logic, I believe we don't need that middle_module logic below any longer. @sloretz do you think this could address the TODO here?

I think that would resolve the TODO on line 68. IIUC in the future it might be possible for there to be more than 3 parts since it's coming from a scoped name in IDL, so you might add a TODO for that

Reply:
done in https://github.com/ros2/ros2cli/pull/242/commits/65a3a218e3a148a4a4714fc4ade7c5acf3ea3f9c